xbmc/pvr/windows/GUIWindowPVRSearch.cpp

```diff
@@ -75,23 +75,23 @@ bool CGUIWindowPVRSearch::FindSimilar(const CFileItemPtr &item)
 
   // construct the search term
   if (item->IsEPG())
-    m_searchfilter.m_strSearchTerm = "\"" + item->GetEPGInfoTag()->Title() + "\"";
+    m_searchfilter.SetSearchTerm("\"" + item->GetEPGInfoTag()->Title() + "\"");
```

Comment:
Since all uses of this method pass the parameter wrapped inside `""`, maybe the setter itself should handle that. At least to me it is not at all obvious that the search term must be wrapped like this.